lib/licensed/sources/swift.rb

```diff
@@ -58,11 +64,12 @@ def homepage_for_url(url)
       end
 
       def package_resolved_file_path
-        File.join(config.pwd, "Package.resolved")
+        File.join(config.pwd, "ios.xcworkspace/xcshareddata/swiftpm", "Package.resolved")
       end
 
-      def swift_package?
-        Licensed::Shell.success?("swift", "package", "describe")
+      def set_derived_data_path
+        build_dir =  JSON.parse(`xcodebuild -showBuildSettings $@ -json`).first.dig("buildSettings", "BUILD_DIR")
+        @derived_data_path ||= build_dir.delete_suffix("/Build/Products")
```

Comment:
Que penses-tu d'enlever l'opérateur `||=` si c'est une méthode qui fait une assignation ?

```suggestion
        @derived_data_path = build_dir.delete_suffix("/Build/Products")
```